docs/_integration-agent/agent-file-folder-listener.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Agent File Folder Listener
+nav_order: 101
+has_children: true
+---
+
+# File Folder Listener
+## Overview
+This mimics the File Folder Listener from Integration Manager.
+The File Folder Listener Service is used to monitor file directories and/or cloud storage buckets/containers for new files. When a new file appears (matching your include/exclude criteria), the associated listener will submit the file to a Job Configuration.
+
+## Basic Configuration
+
+These properties are located in (ProgramDataDirectory)/Actian/FileFolderListener/conf/application.properties **OR**
+/etc/opt/actian/integration-agent/conf/listeners.yml (Linux).
```

Comment:
Could you add in a bit telling the user that these directories have automatic cleanout functionality, so make sure they are isolated from system folders/files.